### PR TITLE
Missing paramiko

### PR DESCRIPTION
**Pull request by jessclmoore**
Testing the example notebooks stellargraph-metapath2vec.ipynb and embeddings-unsupervised-graphsage-cora.ipynb in the embeddings demo directory showed that the paramiko package was required. I added installation instructions to README.md and a requirements.txt to make it easy for users to install. 
Please review.

### CODE REVIEW

**Comment by youph** [comment]
@jessclmoore none of the examples mentioned in the description above require the `paramiko` package. I ran them in an environment without `paramiko` installed, they work fine. Are you sure this is required?

File: demos/embeddings/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+paramiko==2.4.2
```
Comment (youph) [inline demos/embeddings/requirements.txt]: @jessclmoore none of the examples mentioned in the description above require the paramiko package. I ran them in an environment without paramiko installed, they work fine. Are you sure this is required?

Comment (jessclmoore) [reply demos/embeddings/requirements.txt]: @youph Perhaps its related to my system. I received the following warning when running demos/embeddings/stellargraph-metapath2vec.ipynb and demos in
node-classification/node2vec/*
link-prediction/random-walks/*

UserWarning: paramiko missing, opening SSH/SCP/SFTP paths will be disabled. pip install paramiko to suppress

Comment (youph) [reply demos/embeddings/requirements.txt]: Weird, I never saw such warnings before. But despite the warnings the demos work for you without `paramiko` installed, right?

**Comment by adocherty** [comment]
Could I ask what the difference between this PR and the "Missing paramiko ps" PR?

**Pull request closed without merging**